i18n/en.json

```diff
@@ -279,7 +279,7 @@
     "transcoding_accepted_containers_description": "Select which container formats do not need to be remuxed to MP4. Only used for certain transcode policies.",
     "transcoding_accepted_video_codecs": "Accepted video codecs",
     "transcoding_accepted_video_codecs_description": "Select which video codecs do not need to be transcoded. Only used for certain transcode policies.",
-    "transcoding_advanced_options_description": "Options most users should not need to change",
+    "transcoding_advanced_options_description": "Encoding options most users should not need to change",
```

Comment:
There's nothing inherently specific to encoding about this section; it just happens to not have other settings at the moment.

Reply:
Fair; I'll revert this one. I'll drop my rationale for the change below:

I changed it to signpost that it's currently a continuation of encoding options, since I was worried about having some of those options outside the Encoding Options category. At first, I considered nesting Advanced under Encoding Options, but felt the flat structure is nicer aesthetically and less likely for settings to get lost. I also felt it unlikely there would ever be an advanced option not to do with encoding.

Reply:
There was actually a tone-mapping option here until 1.120.0. There could be other niche options in the future as well, like setting frame rate or adding a custom flag to FFmpeg for debugging purposes.

Reply:
Yeah that last one isn't really to do with encoding. Currently, I have (or would have) the other two under encoding tho; that's for later review.